testnets/neuradevnet/chain.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "dneura",
+  "chain_id": "neura_268-1",
+  "bech32_prefix": "neura",
+  "pretty_name": "Neura Devnet",
+  "website": "https://www.neuraprotocol.io/",
+  "description": "Neura is an AI-centric, EVM-compatible Layer 1 blockchain built on the Cosmos SDK. We democratize GPU access and revolutionize AI project funding with IMO’s to advance AI development.",
+  "status": "live",
+  "network_type": "devnet",
+  "node_home": "$HOME/.neurad",
+  "daemon_name": "neurad",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 1,
+  "fees": {
+    "fee_tokens": [
+        {
+            "denom": "atankr"
+        }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+        {
+            "denom": "atankr"
+        }
+    ]
+  },
+  "codebase": {
+    "versions": [
+      {
+        "name": "v0.0.1",
+        "height": 0
+      }
+    ]
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/neuradevnet/images/neura.png"
```

Comment:
Please delete the neura image uploaded to the testnets/neuradevnet/images/ directory.
"image_sync": {
  "chain_name": "neura",
  "base_denom": "atankr"
}